jenkins/check-for-build.jenkinsfile

```diff
@@ -23,6 +23,7 @@ pipeline {
     }
     triggers {
         parameterizedCron '''
+            H 1 * * * %INPUT_MANIFEST=3.3.0/opensearch-dashboards-3.3.0.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards;BUILD_PLATFORM=linux windows;BUILD_DISTRIBUTION=tar rpm deb zip;TEST_MANIFEST=3.3.0/opensearch-dashboards-3.3.0-test.yml;TEST_PLATFORM=linux;TEST_DISTRIBUTION=tar;UPDATE_GITHUB_ISSUE=false
```

Comment:
Added UPDATE_GITHUB_ISSUE=false I believe it is better to enable once we are close to the release.

Reply:
We can revisit this later. Removing it for now. Creating issues because version increment is not merged might also force people to take action.